test/libs/datasets/sources/covid_county_data_test.py

```diff
@@ -14,9 +14,9 @@
 
 def test_fix_tests_and_cases():
     df = read_csv_and_index_fips_date(
-        "fips,date,negative_tests,positive_tests,total_tests,cases\n"
-        "97123,2020-04-01,9,1,10,1\n"
-        "97123,2020-04-02,,,20,2\n"
+        "fips,state,date,negative_tests,positive_tests,total_tests,cases\n"
+        "97123,,2020-04-01,9,1,10,1\n"
+        "97123,,2020-04-02,,,20,2\n"
```

Comment:
`CovidCountyDataDataSource.synthesize_test_metrics()` now relies on the `state` column in order to do the disabling...